nova_vm/src/ecmascript/syntax_directed_operations/scope_analysis.rs

```diff
@@ -150,7 +150,7 @@ impl<'a> LexicallyDeclaredNames<'a> for Statement<'a> {
                     ExportDefaultDeclarationKind::FunctionDeclaration(decl) => decl.bound_names(f),
                     ExportDefaultDeclarationKind::ClassDeclaration(decl) => decl.bound_names(f),
                     ExportDefaultDeclarationKind::TSAsExpression(_)
-                    | ExportDefaultDeclarationKind::TSEnumDeclaration(_)
+                    // | ExportDefaultDeclarationKind::TSEnumDeclaration(_)
```

Comment:
**nitpick:** Would be better to simply remove these variants, rather than commenting them out.

Reply:
done :)